web/src/services/Api.ts

```
@@ -70,24 +75,84 @@ function toErrorResponse<T>(resp: AxiosResponse<ServiceResponse<T>>): ErrorRespo
+    if (valueRetryAfter) {
+        if (/^\d+$/.test(valueRetryAfter)) {
+            const retryAfter = parseFloat(valueRetryAfter);
+
+            if (Number.isNaN(retryAfter)) {
+                throw new Error("Header Retry-After has an invalid number value");
+            }
+
+            return retryAfter;
+        } else {
+            const date = new Date(valueRetryAfter);
+            if (isNaN(date.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)